coalib/bearlib/aspects/Smell.py

```diff
@@ -142,15 +142,15 @@ def func(a, b, c, d, e, f, g, h, i, j, k):
         example_language = 'python'
         importance_reason = """
         Refactoring such methods and functions may reveal some duplicated
-        code, make the code shorter, easy to read and maintain
+        code, make the code shorter, easy to read and maintain.
         """
         fix_suggestions = """
         Usually passing objects to functions or methods instead of passing
         their attributes, if possible grouping some paramaters into a single
-        object, splitting up methods... can fix this.
+        object, splitting up methods etc. can fix this.
```

Comment:
this whole paragraph makes no sense, has no real predicate, try rewriting it a bit.